github/personal-website
# Make stargazers and forks stick to bottom of repo cards

## Goal of this PR

This pull request makes repository cards behave like the repository cards that can be seen on GitHub, by having the bottom row (with stargazers and forks) stick to the bottom.

Here is an example of what it looks like on `master`

<img width="369" alt="screenshot 2019-02-23 at 05 59 38" src="https://user-images.githubusercontent.com/6976628/53281908-802e8100-3730-11e9-9583-5b9393fdb28a.png">

And of the layout proposed in this PR

<img width="369" alt="screenshot 2019-02-23 at 06 00 54" src="https://user-images.githubusercontent.com/6976628/53281910-845a9e80-3730-11e9-8b3f-437d3753c175.png">

In order to achieve that, three minor changes were added:

* The repo card itself gets a new `position: relative` CSS property
* The `div` that wraps the stargazers and forks gets the `position: absolute` and `bottom: 10px` CSS properties
* The `div` that wraps the description has its `mb-2` (margin bottom 2) class replaced with a `mb-4` in order to compensate for the bottom row now being absolute and not pushing the repository card's height anymore.

Fixes #25 

## How to test it

* Checkout this branch
* `bundle exec jekyll serve`
* Visit `http://127.0.0.1:4000` and ensure the cards are behaving properly, with different viewport sizes (from desktop to mobile)
